5.6/Dockerfile.rhel7

```diff
@@ -0,0 +1,61 @@
+FROM rhel7
+
+# This image provides an Apache+PHP environment for running PHP
+# applications.
+
+MAINTAINER docker@softwarecollections.org
+
+EXPOSE 8080
+
+ENV PHP_VERSION=5.6 \
+    PATH=$PATH:/opt/rh/rh-php56/root/usr/bin
+
+LABEL io.k8s.description="Platform for building and running PHP 5.6 applications" \
+      io.k8s.display-name="Apache 2.4 with PHP 5.6" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,php,php56"
+
+LABEL k8s.io/description="Platform for building and running PHP 5.6 applications" \
+      k8s.io/display-name="Apache 2.4 with PHP 5.6" \
+      openshift.io/expose-services="8080:http" \
+      openshift.io/tags="builder,php,php56"
```

Comment:
I'd say so. if it is just for searching, somebody may try to find this image by asking for SCL name.